config/testdata/config.yml

```diff
@@ -50,6 +50,7 @@ ios:
   key_type: "pem" # could be pem, p12 or p8 type
   password: "" # certificate password, default as empty string.
   production: false
+  max_concurrent_pushes: 100
```

Comment:
maybe set the initial value as zero (default `runtime.NumCPU()`) like `worker_num`

Reply:
I wouldn't do that as its not CPU constrained (There will be a limit but its very high, at which point you will benefit from spreading it across multiple clients and thus multiple cores).

Im running several 1000 from my macbook with no issue what so ever. APNS documentation said you should easily be expecting 9000 per second. (granted its not quite the same as concurrent connections)

If you really want to I change it to 1 (0 would prevent it from working), but I was getting as little as 3 per second when there was no concurrency.